src/components/Footer/index.tsx

```diff
@@ -53,13 +54,13 @@ const Footer = () => {
             </a>
           </li>
           <li>
-            <a href='https://tech.deriv.com/' target='blank' className={styles.Link}>
-              <LabelPairedArrowUpRightSmRegularIcon /> <Translate>Deriv Tech</Translate>
+            <a href='https://deriv.com/derivtech' target='blank' className={styles.Link}>
```

Comment:
can we change the target to `_blank` instead of `blank`?
```suggestion
            <a href='https://deriv.com/derivtech' target='_blank' className={styles.Link}>
```